src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/IconTintColor/IconTintColorEffectRouter.android.cs

```diff
@@ -51,21 +53,21 @@ void ApplyTintColor()
 
 		void ClearTintColor()
 		{
-			try
+			// Because of a XF bug: https://github.com/xamarin/Xamarin.Forms/issues/13889
+			if (!Control.IsAlive())
 			{
-				switch (Control)
-				{
-					case ImageView image:
-						image.ClearColorFilter();
-						break;
-					case Button button:
-						foreach (var drawable in button.GetCompoundDrawables())
-							drawable?.ClearColorFilter();
-						break;
-				}
+				return;
```

Comment:
Instead of an empty return, can you add a log message here? Something that says that the Control is disposed.

Reply:
Since we are on Android using https://developer.android.com/reference/android/util/Log#e(java.lang.String,%20java.lang.String) ?